test/fixtures/permission/fs-write.js

```diff
@@ -11,6 +11,7 @@ const regularFolder = process.env.ALLOWEDFOLDER;
 const regularFile = process.env.ALLOWEDFILE;
 const blockedFolder = process.env.BLOCKEDFOLDER;
 const blockedFile = process.env.BLOCKEDFILE;
+const blockedFileURL = new URL('file://' + process.env.BLOCKEDFILE);
```

Comment:
This would fail if the `BLOCKEDFILE` path contains any URL-special chars
```suggestion
const blockedFileURL = require('url').pathToFileURL(process.env.BLOCKEDFILE);
```